builtin/game/chatcommands.lua

```diff
@@ -813,7 +813,7 @@ core.register_chatcommand("time", {
 })
 
 core.register_chatcommand("days", {
-	description = "Display day count",
+	description = "Display day count since world creation",
```

Comment:
'Show' instead? Or maybe use 'display' instead of 'show' elsewhere for consistency?